glutton/conntrack.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
I try to only have running things in the main package. All functions should go into the glutton package.